docker-hub/repos.md

```diff
@@ -53,6 +53,35 @@ images to this repository.
 
 3. Enter the name of the repository to confirm the deletion and click **Delete**.
 
+## Consolidating a repository
+
+### Personal to personal
+
+When consolidating personal repositories, you can pull private images from the initial repository and push them into another repository that is owned by you. To avoid losing your private images, perform the following steps listed below.
+
+1. Navigate to Docker Hub, create a Docker ID and select the personal subscription.
+2. Using `docker login` from the CLI, log in using your original Docker ID and pull your private images.
+3. Tag your private images with your newly created Docker ID using:
+`docker tag namespace1/docker101tutorial ‘new_namespace/docker101tutorial`
+4. Using `docker login` from the CLI, log in with your newly created Docker ID, and push your newly tagged private images to your new Docker ID namespace.
+`docker push <new_namespace>/docker101tutorial`
+5. The private images that existed in your previous namespace are now available in your new Docker ID namespace.
+
+### Personal repo to an organization
+
+To avoid losing your private images, you can pull your private images from your personal namespace and push them to an organization that is owned by you.
+
+1. Navigate to [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and select **Organizations**.
+2. Select the applicable organization team and verify that your user account is a member of the organization.
```

Comment:
```suggestion
2. Select the applicable organization and verify that your user account is a member of the owners team.
```

Reply:
@craig-osterhout The owners team or just the org? If their moving from personal repo > org?

Reply:
@jerae-duffin Good point. I haven't tested it, but above step 1 it says the organization must be owned by you. Maybe that isn't necessary.
`To avoid losing your private images, you can pull your private images from your personal namespace and push them to an organization that is owned by you.`

Reply:
Thank you!